naucse/templates/_base.html

```diff
@@ -62,7 +62,39 @@
 
         <script type="text/javascript" src="{{ url_for('static', filename='js/solutions.js') }}"></script>
 
-        {% block footer %}{% endblock %}
+        <div class="footer">
+            <div class="container">
+                <div class="row">
+                    <div class="col">
+                        <h2>Zapoj se</h2>
+                        <p>
+                            {% if lesson is defined %}
+                                Uprav tuto stránku na <a href="https://github.com/pyvec/naucse.python.cz/blob/master/lessons/{{ page }}.{{ page.style }}">GitHubu</a>
+                            {% else %}
+                                Uprav tuto stránku na <a href="https://github.com/pyvec/naucse.python.cz">GitHubu</a>
```

Comment:
Could the link be determined on anther level (view or model) and only used here as follows?

```
Uprav tuto stránku na <a href="{{ page.github_link }}">GitHubu</a>
```